### ISSUE TITLE
Limit Polr to Custom URLs / Custom Domains?

### ISSUE THREAD

**Issue by kcseb** [cydrobolt/polr#565]
I could not find a specific issue regarding this, and asking on the IRC is a bit pointless as there is only three people, including myself, in the channel... 

Is it possible to limit Polr to specific domains / URLs? For example. you can only shorten domains that include "exampleDomain.com" 

(so, "exampleDomain.com/thingy.html" -> "polr.cat/xDnnH" BUT exampleWebby.net -> "Sorry, you can not shorten this domain here.") 

Thank you!

**Comment by ceisma** [comment]
Yes, I'm looking for the same feature! I hope that they will include this. Perhaps that you can add your preferred domain(s) to a "restricted to" list in the setup or something like that.  :)

**Comment by ahangarha** [comment]
I am willing to work on this issue. Will start it if the maintainer approves it till end on June 2021.
